spec/src/main/asciidoc/java-api.adoc

```diff
@@ -171,17 +171,15 @@ Health check procedures are CDI beans, therefore, they can also be defined with
 class MyChecks {
 
   @Produces
-  @ApplicationScoped
   @Liveness
   HealthCheck check1() {
-    return () -> HealthStatus.state(getMemUsage() < 0.9);
+    return () -> HealthCheckResponse.state(getMemUsage() < 0.9);
```

Comment:
this would not compile, can you please add `.named("live")` or similar and `.build()`, please?